stdlib/test/collections/test_list.mojo

```diff
@@ -741,6 +741,19 @@ def test_list_mult():
     assert_equal(len(List[Int](1, 2, 3) * 0), 0)
 
 
+def test_list_contains():
+    var x = List[Int](1, 2, 3)
+    assert_equal(0 in x, False)
```

Comment:
**Suggestion** Can you rewrite these to use `assert_true` and `assert_false`, respectively, please?  E.g.

```mojo
assert_false(0 in x)
assert_true(1 in x)
assert_false(4 in x)
```